### PR TITLE
link fixed

### LINKED ISSUE #1057: docs: Wrong link in image examples
http://bem.info/m/_/20zuV2gLiyoaqrHv9LoBRTmB0Ow.svg leads to 404 at
https://github.com/bem/bem-components/blob/v2/common.blocks/image/image.en.md (same in Russian version)

### PR DESCRIPTION
**Pull request by innabelaya**
closes #1057

### CODE REVIEW

File: common.blocks/image/image.en.md [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 ```bemjson
 {
-    block : 'image',
-    url : 'http://bem.info/m/_/20zuV2gLiyoaqrHv9LoBRTmB0Ow.svg',
-    width : 100,
-    height : 100,
-    alt : 'BEM',
-    title : 'Find us on bem.info'
+    block: 'image',
+    url: 'https://yadi.sk/d/HEWCF7VVcFQmJ',
```
Comment (tadatuta) [inline common.blocks/image/image.en.md]: this link won't work :(
not sure if it's permanent but right now direct link to the image is https://leto19e.storage.yandex.net/rdisk/b54f9103b823bc4736f3f3dc28e6d2da/mpfs/wWENDFjI_yyeYelosYTzeTZOexYDJuWgJNFA2sR-k4SzbSTsxmU553Sy6GevtJoX924mUhuVNiktKwt4_olT1g==?uid=0&filename=logo_theme_spots.png&disposition=inline&hash=&limit=0&content_type=image%2Fpng&rtoken=25c77bc2e583dac822140b042a9f4217&rtimestamp=544a0799&force_default=no (still I don't think it's the good idea).

let's just update the link to new hash. as we are aware of the problem I hope we won't mess everything again :)

File: common.blocks/image/image.en.md [hunk 2]
```diff
@@ -4,12 +4,12 @@
 
 ```bemjson
 {
-    block : 'image',
```
Comment (tadatuta) [inline common.blocks/image/image.en.md:7]: why do you remove spaces before semicolon? that's the part of our codestyle.

Comment (innabelaya) [inline common.blocks/image/image.en.md:7]: Spaces are returned to their places now :)

File: common.blocks/image/image.ru.md [hunk 3]
```diff
@@ -4,13 +4,12 @@
...
 
-
 ```bemjson
 {
     block : 'image',
-    url : 'http://bem.info/m/_/20zuV2gLiyoaqrHv9LoBRTmB0Ow.svg',
+    url : 'http://bem.info/m/_/IvuRSFeBStMXdBhBrBvKagSd5sE.svg',
     width : 100,
-    height : 100,
+    height :  100,
```
Comment (tadatuta) [inline common.blocks/image/image.ru.md]: even more spaces then before! :)

Comment (innabelaya) [inline common.blocks/image/image.ru.md]: oooops :(

**Comment by innabelaya** [comment]
hope, now it is correct :)

**Comment by tadatuta** [comment]
merged but let's use protocol-less links (started with `//`) in the future

**Pull request merged**